doc/user/content/installation/install-on-azure/appendix-deployment-guidelines.md

```diff
@@ -10,21 +10,35 @@ menu:
 
 ## Recommended instance types
 
-- ARM-based CPU
-- 1:8 ratio of vCPU to GiB memory (if spill-to-disk is not enabled)
-- 1:16 ratio of vCPU to GiB local instance storage (if spill-to-disk is enabled)
+As a general guideline, we recommend:
 
+- Processor Type: ARM-based CPU
 
-## CPU affinity
+- Sizing: 2:1 disk-to-RAM ratio with spill-to-disk enabled.
 
-It is strongly recommended to enable the Kubernetes `static` [CPU management policy](https://kubernetes.io/docs/tasks/administer-cluster/cpu-management-policies/#static-policy).
-This ensures that each worker thread of Materialize is given exclusively access to a vCPU. Our benchmarks have shown this
-to substantially improve the performance of compute-bound workloads.
+### Recommended Azure VM Types with Local NVMe Disks
 
-## TLS
+When operating on Azure in production, we recommend [Epdsv6
+sizes](https://learn.microsoft.com/en-us/azure/virtual-machines/sizes/memory-optimized/epdsv6-series?tabs=sizebasic#sizes-in-series)
+Azure VM Types with Local NVMe Disk:
 
-When running with TLS in production, run with certificates from an official
-Certificate Authority (CA) rather than self-signed certificates.
+| VM Size            | vCPUs | Memory  | Ephemeral Disk | Disk-to-RAM Ratio |
+| ------------------ | ----- | ------- | -------------- | ----------------- |
+| Standard_E2pds_v6  | 2     | 16 GiB  | 75 GiB         | ~4.7:1           |
+| Standard_E4pds_v6  | 4     | 32 GiB  | 150 GiB        | ~4.7:1           |
+| Standard_E8pds_v6  | 8     | 64 GiB  | 300 GiB        | ~4.7:1           |
+| Standard_E16pds_v6 | 16    | 128 GiB | 600 GiB        | ~4.7:1           |
+| Standard_E32pds_v6 | 32    | 256 GiB | 1,200 GiB      | ~4.7:1           |
+
+{{< important >}}
+
+These VM types provide **ephemeral** local NVMe SSD disks. Since data is lost
+when the VM is stopped or deleted, they are best suited for **temporary or
+performance-sensitive data** managed by Materialize.
```

Comment:
Nit: Materialize only uses disk for ephemeral data, all other data is in block storage, so the "best suited" part doesn't make much sense.

Reply:
👍 Thanks much!  ❤️  Will just end with "Data is lost when the VM is stopped or deleted." then.